blackberggroup/VA-404
# Feature/jls 6102298803/new 404 stand alone repo

## What type of PR is this? (check all applicable)


## Description
From OHIL repo, create new stand-alone 404 page repo.
Cleaned up repo: rm unused OHIL and iEX assets and pages; edited deploy paths and exclusions; duplicated _site/index.html as _site/404.html; changed paths to use assets not assets-3-5; rm assets-3-5; menu dropdown arrow overrides.
 
## Related Tickets & Documents
- Monday Task #6102298803

## QA Instructions, Screenshots, Recordings
New 404 page based on those from [NIH](https://www.nih.gov/abc) and [NCI](https://www.cancer.gov/abc). Feel free to suggest copy edits or additions.
Please look at deploy files and confirm paths and exclusions (don't want to overwrite OHIL's index.html)

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_


## Are there any post deployment tasks we need to perform?
- Add GA code on production
- Proper meta tags utilized
